src/components/tile/tile.stories.mdx

```diff
@@ -0,0 +1,273 @@
+        orientation="horizontal"
+        padding="M"
+        pixelWidth={0}
+        width={75}
+        >
+        <Content
+            align="left"
+            as="primary"
+            bodyFullWidth={false}
+            inline={false}
+            title="Test Title One"
+            width={40}
+        >
+            Test Body One
+        </Content>
+        <Content
+            align="left"
+            as="primary"
+            bodyFullWidth={false}
+            inline={false}
+            title="Test Title Two"
+            width={80}
+        >
+            Test Body Two
+        </Content>
+        <Content
+            align="left"
+            as="primary"
+            bodyFullWidth={false}
+            inline={false}
+            title="Test Title Three"
+            width={120}
+        >
+            Test Body Three
+        </Content>
+        </Tile>
+    </Story>
+</Preview>
+
+## Tile with Inline Styling
+The Tile component can also have inline styling applied to the content.
+
+<Preview>
+    <Story name ='with Inline'>
+        <Tile
+        as="tile"
+        orientation="horizontal"
+        padding="M"
+        pixelWidth={0}
+        width={0}
+        >
+        <Content
+            align="left"
+            as="primary"
+            bodyFullWidth={false}
+            inline={true}
+            title="Test Title One"
+            width={80}
+        >
+            Test Body One
+        </Content>
+        <Content
+            align="left"
+            as="primary"
+            bodyFullWidth={false}
+            inline={true}
+            title="Test Title Two"
+            width={80}
+        >
+            Test Body Two
+        </Content>
+        <Content
+            align="left"
+            as="primary"
+            bodyFullWidth={false}
+            inline={true}
+            title="Test Title Three"
+            width={80}
+        >
+            Test Body Three
+        </Content>
+        </Tile>
+    </Story>
+</Preview>
+
+## Props of Tile
+<Props of={ Tile } />
+
+# Tile with Definition List
+The Definition List component should only be used inside of the Tile component. The purpose of the Definition List component is to mimic
+the same tags used in HTML5 but instead they are React elements. 
+
+## Quick Start
+```jsx
+import Tile from 'carbon-react/lib/components/tile';
+import {Dl, Dt, Dd} from 'carbon-react/lib/components/definition-list';
+```
+<Preview>
+  <Story name ='Tile with Definition List - default'>
+    <Tile width={95}>
+      <Dl>
+        <Dt>Drink</Dt>
+        <Dd>Coffee</Dd>
+        <Dt>Brew Method</Dt>
+        <Dd>Stove Top Moka Pot</Dd>
+        <Dt>Brand of Coffee</Dt>
+        <Dd>Magic Coffee Beans</Dd>
+        <Dt>Website</Dt>
+        <Dd><Link href='www.sage.com'>Magic Coffee Beans' Website</Link></Dd>
+        <Dt>Email</Dt>
+        <Dd><Link href='magic@coffeebeans.com'>magic@coffeebeans.com</Link></Dd>
+        <Dt>'Main and Registered Address'</Dt>
```

Comment:
![image](https://user-images.githubusercontent.com/14963680/92131159-87424f00-edfd-11ea-93a4-0a5f49b576df.png)
I think it may be an accident that there's quotes around this?

Reply:
Yeah. It's because they were strings when I had the `label` prop. Copied them and left this single quotes around this. Cheers for spotting that 👍

Reply:
✅